README.md

```diff
@@ -391,40 +385,11 @@ const data = await response.json();
 console.log(data)
 ```
 
-### Post with form-data (detect multipart)
+node-fetch also supports spec-compliant FormData implementations such as [formdata-polyfill](https://www.npmjs.com/package/formdata-polyfill) and [formdata-node](https://github.com/octet-stream/form-data):
 
 ```js
-const fetch = require('node-fetch');
-const FormData = require('form-data');
-
-const form = new FormData();
-form.append('a', 1);
-
-const response = await fetch('https://httpbin.org/post', {method: 'POST', body: form});
-const data = await response.json();
-
-console.log(data)
-
-// OR, using custom headers
-// NOTE: getHeaders() is non-standard API
-
-const options = {
-	method: 'POST',
-	body: form,
-	headers: form.getHeaders()
-};
-
-const response = await fetch('https://httpbin.org/post', options);
-const data = await response.json();
-
-console.log(data)
-```
-
-node-fetch also supports spec-compliant FormData implementations such as [form-data](https://github.com/form-data/form-data) and [formdata-node](https://github.com/octet-stream/form-data):
-
-```js
-const fetch = require('node-fetch');
-const FormData = require('formdata-node');
+import fetch from 'node-fetch';
+import FormData from 'formdata-node';
```

Comment:
formdata-node does not have default export

```diff
- import FormData from "formdata-node"
+ import {FormData} from "formdata-node"
```

Reply:
@jimmywarting should we fix this before merging?

Reply:
sure